src/modules/rover_differential/RoverDifferentialGuidance/RoverDifferentialGuidance.cpp

```diff
@@ -96,6 +96,16 @@ void RoverDifferentialGuidance::computeGuidance(const float vehicle_yaw, const f
 								_param_rd_max_decel.get(), distance_to_curr_wp, 0.0f);
 					desired_forward_speed = math::constrain(desired_forward_speed, -_max_forward_speed, _max_forward_speed);
 				}
+
+			} else if (_param_rd_max_jerk.get() > FLT_EPSILON && _param_rd_max_decel.get() > FLT_EPSILON
+				   && _param_rd_miss_spd_gain.get() > FLT_EPSILON) {
+				const float speed_reduction = math::constrain(_param_rd_miss_spd_gain.get() * math::interpolate(
```

Comment:
Could it be an option to use RD_TRANS_DRV_TRN for the slow down instead of introducing this new param RD_MISS_SPD_GAIN?

Reply:
`RD_TRANS_DRV_TRN` is used in the slow down effect to decide if the rover comes to a stop at the current waypoint and then turn on the spot, or if it only needs to slow down. `RD_MISS_SPD_GAIN` is a parameter to tune how much it slows down. They have very different purposes so I don't really see how I could combine them.